packages/css/src/Footer/Footer.client.scss

```diff
@@ -0,0 +1,178 @@
+@use "../common/common.client.scss" as common;
+
+.af-footer__menuLinks--display {
+  height: auto;
+  padding: 0 2rem;
+  overflow: hidden;
+  text-transform: uppercase;
+  transition:
+    background 250ms,
+    padding 125ms;
+}
+
+.af-footer__menuLinks {
+  height: 0;
+  padding: 0;
+  overflow: hidden;
+  text-transform: uppercase;
+  background: transparent;
+  transition:
+    background 0ms,
+    padding 0ms;
+}
+
+.af-footer__iconTrigger--display {
+  display: flex;
+  align-self: center;
+  justify-content: center;
+  transform: rotate3d(1, 0, 0, 180deg);
+  transition: transform 250ms ease-in-out;
+}
+
+.af-footer__iconTrigger {
+  display: flex;
+  align-self: center;
+  justify-content: center;
+  transform: rotate3d(0, 0, 0, 180deg);
+  transition: transform 250ms ease-in-out;
+}
+
+.af-footer__linkItem {
+  display: block;
+  padding: 1.2rem common.$spacing-u3;
+  text-decoration: none;
+  color: common.$color-white;
+}
+
+.af-footer__footerMenuIcons {
+  display: flex;
+  padding: common.$spacing-u2 common.$spacing-u1;
+  border: 1px solid rgba(255, 255, 255, 20%);
+  border-width: 1px 0;
+  flex-direction: row;
+  justify-content: center;
+  gap: common.$spacing-u3;
+}
+
+.af-footer__menuIconLinks {
+  align-self: center;
+  color: common.$color-white;
+}
+
+.af-footer {
+  z-index: 100;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  color: common.$color-white;
+  background-color: common.$color-oceanBlue;
+}
+
+.af-footer__menuAboutTrigger {
+  all: unset;
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  padding: common.$spacing-u2 common.$spacing-u4;
+  justify-content: space-between;
+}
+
+.af-footer__menuAboutTriggerText {
+  font-weight: bold;
+  text-transform: uppercase;
+  color: common.$color-white;
+}
+
+.af-footer__footerBottom {
+  padding: common.$spacing-u1;
+}
+
+.af-footer__textCopyright {
+  display: flex;
+  justify-content: center;
+  font-size: 0.875rem;
+}
+
+.af-footer__icon {
+  width: 16px;
+  height: 16px;
+  align-self: center;
+  fill: common.$color-white;
+}
+
+@media only screen and (width >= 1024px) {
```

Comment:
on pourrait créer des token sass avec toutes les valeurs des grilles dans le fichier commons

Reply:
ah et bien elle existe déjà $screen-size-m

Reply:
je veux bien mais quand j'essaye ca me dit :
![image](https://github.com/AxaFrance/design-system/assets/35838907/7997bbf8-375d-44bf-ab16-4518067cc686)

Reply:
tu as essayé en faisant de l'interpolation ? 
@media only screen and (width >= #{common.$screen-size-m}) {

}

Reply:
ca passe comme ca, merci